components/Header.js

```diff
@@ -20,8 +20,7 @@ const StyledHeader = styled.header`
   text-align: center;
   font-family: var(--fontfamily-special);
   font-size: 1.3rem;
-  position: fixed;
```

Comment:
Without `position: fixed` the height of the header increases a bit.

Reply:
the height of the header is now defined in the AppContainer styling. Ich reduced it to get the same height as before.